src/components/PageWrap.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { polyfillServiceUrl } from '../util/browserPolyfill';
```

Comment:
In working to make things more modular in MWP, I've realized that almost everything in `src/util/` would be better placed somewhere else (usually closer to the scripts that consume the utils).

In this case, I would recommend a new `/src/components/util/` (or `/src/components/pageUtils/`) directory to put `browserPolyfill.js` and `newrelicBrowser.js` - since those are only used in this component, that organization keeps those modules better 'hidden' from the rest of the application.

I have a modularization ticket to move all the PlatformApp/BrowserApp-related components to their own module, so this code will move along with that sometime soon, so keeping everything close together now will help with that.

---

thinking about this a bit more, since there are quite a few files that need to move around with `PageWrap`, I think I'd recommend making a single directory that contains _all_ the related scripts, e.g. `src/components/page-wrap`, so that the related files can be moved around together easily - no need to spread things among `src/components/`, `src/util/` and `src/asset/`, which makes the 'system' unwieldy